# List of joint positions
float64[] joints
